chart/templates/post-delete-hook-clusterRole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.postDelete.enabled }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Values.postDelete.name }}
+  labels:
+{{ include "rancher.labels" . | indent 4 }}
```

Comment:
nit:
```suggestion
  labels: {{ include "rancher.labels" . | nindent 4 }}
```

Reply:
same comment for everywhere else this is used.

Reply:
fixed all occurrences.